examples/Makefile-nrf52840

```diff
@@ -79,6 +79,10 @@ COMMONCFLAGS                   := \
     -I$(CONFIG_FILE_PATH)         \
     $(NULL)
 
+ifeq ($(CERT_LOG),1)
+COMMONCFLAGS += -DOPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT=1
```

Comment:
@abtink, note that this PR ties `OPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT` to `--enable-cert-log` configure option.  We could add to the project include, but we would also need to make sure `include/openthread/config.h` is included in front.